Presentation.Web/app/utility/kendoGridLauncher.ts

```diff
@@ -674,7 +682,9 @@ module Kitos.Utility.KendoGrid {
 
         saveGridForOrganization() {
             if (confirm(`Er du sikker på at du vil gemme nuværende kolonneopsætning af felter som standard til ${this.user.currentOrganizationName}`)) {
-                this.gridState.saveGridOrganizationalConfiguration(this.gridBinding.mainGrid, this.overviewType);
+                return this.gridState
+                    .saveGridOrganizationalConfiguration(this.gridBinding.mainGrid, this.overviewType)
```

Comment:
something is broken here now:
After saving, this happens:
![image](https://user-images.githubusercontent.com/13116889/193784425-c94f7686-1c73-44a2-b037-1493bdc1c1c0.png)

Reply:
@a-naskret I committed at fix for this